### PR TITLE
Some DocString and type annotations

### PR DESCRIPTION
**Pull request by Carreau**
While I'm getting familiar with the code.

### CODE REVIEW

File: stack_data/core.py [hunk 1]
```diff
@@ -178,19 +206,17 @@ def render_with_markers(self, markers, strip_leading_indent=True):
         return ''.join(parts)
 
 
-Range = namedtuple('Range', 'start end data')
-
 
-def markers_from_ranges(ranges, converter):
+def markers_from_ranges(ranges:List[Range], converter)->List[Marker]:
```
Comment (alexmojaki) [inline stack_data/core.py]:
```suggestion
def markers_from_ranges(ranges: List[Range], converter) -> List[Marker]:
```

Please keep the code PEP8 compliant.

**Comment by alexmojaki** [comment]
I appreciate the effort but I think leave out the docstrings. I think I'll find it easier to write them myself once the code is ready for release, which is far away.

**Comment by Carreau** [comment]
> Please keep the code PEP8 compliant.

Yeah, I haven't yet assimilated pep8 rules from types... I'm considering using black for my projects to not have to think about that. 




> I appreciate the effort but I think leave out the docstrings.

Removed.

**Comment by alexmojaki** [comment]
There were still some docstrings and PEP8 issues, plus other difficulties. Again, it was easier to just do it myself.

**Pull request closed without merging**